src/test_priorty.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+"""Testing module for the Heap classes."""
+
+from priorty import PriortyQ
+import pytest
+import random
+
+
+# ------------------Heap Initialization Tests--------------------
+# [x] Test default initialization of a priortyQ is a priortyQ.
+
+
+def test_pq_initalization():
+    """Test default initialization of a priortyQ is a priortyQ."""
+    pq = PriortyQ()
+    assert isinstance(pq, PriortyQ)
+
+
+# ------------------Insert Method Tests--------------------
+# [x] Insert one value into the heap check that value in heap.
+# [x] Insert one value into the heap check at position 0.
+# [x] Insert, insert bigger, greater priorty at position 0.
+# [x] Insert, insert bigger, less priorty at position 1.
+
+def test_pq_insert_one_item():
+    """Insert one value into the heap check that value in priorty queue."""
+    pq = PriortyQ()
+    pq.insert("item", 1)
+    assert (1, "item") in pq._queue._heap
+
+
+def test_pq_insert_one_item_check_position():
+    """Insert one value into the heap check at position 0."""
+    pq = PriortyQ()
+    pq.insert("item", 1)
+    assert pq._queue._heap[0] == (1, "item")
+
+
+def test_pq_insert_two_items_check_position0():
+    """Insert one value into the heap check at position 0."""
+    pq = PriortyQ()
+    pq.insert("item", 1)
+    pq.insert("item2", 2)
+    assert pq._queue._heap[0] == (2, "item2")
+
+
+def test_pq_insert_two_items_check_position1():
+    """Insert one value into the heap check at position 0."""
+    pq = PriortyQ()
+    pq.insert("item", 1)
+    pq.insert("item2", 2)
+    assert pq._queue._heap[1] == (1, "item")
```

Comment:
Your write-up of the priority queue description says that "negative priorities" would be of a lower priority than the default but I don't see you testing that anywhere.  Are you sure it is working?